source/content/Telco_20230213.md

```diff
@@ -58,7 +58,7 @@ Conclusion: processing results shall be stored separately, but their structure c
 SB: documentation is important, NIAC shall not accept poor definition files  
 SB: NXdata base class also has offset and scaling_factor Fields  
 BW: MK added @offset, @scaling, @transformation without documentation in the nxdl file, but refferrng to 2010 NIAC. See:  
-[minutes of 2010 NIAC](https://github.com/nexusformat/wiki/blob/master/content/NIAC2010.html) and  
+[minutes of 2010 NIAC](https://github.com/nexusformat/wiki/blob/master/source/content/NIAC2010.html) and 
```

Comment:
this link is a 404, the html files are not part of the repo so this should likely point to a relative link such as (NIAC2010.html)

Reply:
Thanks for the catch